sagar/data/Lab4Visual.ipynb

```diff
@@ -0,0 +1,1121 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import plotly.express as px
</pre><p>No need for this, You can achieve this with standard matplotib (nested pie chart) refer to index.ipynb in main/lab-4-visualization-of-one-dimensional-data</p><p> </p><p>Also, no output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>